src/savegame_json.cpp

```diff
@@ -728,6 +730,11 @@ void player::deserialize(JsonIn &jsin)
 
     data.read( "stamina", stamina);
     data.read( "move_mode", move_mode );
+    // TODO: The if clause can be omitted after some time, just there for old savegames
+    if ( !data.read( "selected_move_mode", selected_move_mode ) ) {
+       selected_move_mode = move_mode;
+    }
+    data.read( "move_mode",  moved );
```

Comment:
You're reading the same JSON value as above (line 732), is this intentional?

Reply:
No not intentional, thank you. Should read the "moved" value.